packages/webdriverio/src/index.js

```diff
@@ -4,6 +4,7 @@ import { validateConfig, wrapCommand, runFnInFiberContext, detectBackend } from
 import MultiRemote from './multiremote'
 import { WDIO_DEFAULTS } from './constants'
 import { getPrototype } from './utils'
+import path from 'path'
```

Comment:
Also this import needs to move up to the first line

Reply:
fixed